src/main/java/de/tum/p2p/onion/auth/NettyRemoteOnionAuth.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.p2p.onion.auth;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class NettyRemoteOnionAuth implements OnionAuthorizer {
```

Comment:
duplicate btw
this : `de.tum.p2p.onion.auth.NettyRemoteOnionAuth`
that: `de.tum.p2p.onion.auth.netty.NettyRemoteOnionAuth`